tools/configure.sh

```diff
@@ -168,16 +164,25 @@ fi
 
 src_config=${configpath}/defconfig
 dest_config="${TOPDIR}/.config"
+backup_config="${TOPDIR}/defconfig"
 
 if [ ! -r ${src_config} ]; then
   echo "File ${src_config} does not exist"
   exit 5
 fi
 
-if [ ${skip} != 1 ] && [ -r ${dest_config} ]; then
-  echo "Already configured!"
-  echo "Do 'make distclean' and try again."
-  exit 6
+if [ -r ${dest_config} ]; then
+  if cmp -s ${src_config} ${backup_config}; then
+    exit 0 # No any config change
+  fi
+
+  if [ "X${enforce}" = "Xy" ]; then
+    make -C ${TOPDIR} $* distclean 1>/dev/null
```

Comment:
@xiaoxiang781216 Do you prefer running distclean quietly? For a second I thought my system was hung.
(Maybe it's just my slow laptop?)

Reply:
> @xiaoxiang781216 Do you prefer running distclean quietly? For a second I thought my system was hung.
> (Maybe it's just my slow laptop?)

I will remove "1>/dev/null" if -d is specified.